config.js

```diff
@@ -35,6 +35,9 @@ const defaultDbUrl = 'postgres://' + pguser + ':' + pgpass + '@' + pghost +
 const ipWhitelist = pe.IP_WHITELIST || '[[0.0.0.0,255.255.255.255]]';
 const iplist = configUtil.parseIPlist(ipWhitelist);
 
+// Check for timed-out samples every 30 seconds if not specified in env var
+const DEFAULT_CHECK_TIMEOUT_INTERVAL_MILLIS = 30000;
+
```

Comment:
Heh I went back on what I said... I realized it is just easier for people to understand what they're looking at in the code if it's a round number :)